src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -56,18 +57,23 @@ public void verifyMobileTopLeaderboard() {
     removeElementIfPresent(SMART_BANNER_SELECTOR); // Only works for WikiaMobile
 
     wait.forElementVisible(presentLeaderboard);
+    if (isElementOnPage(By.cssSelector(INTERSTITIAL_AD_OPENED_SELECTOR))) {
+      PageObjectLogging.logWarning("Special ad", "Interstitial ad detected");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Special ad" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)